app/src/main/res/layout/drink_card.xml

```diff
@@ -35,86 +34,80 @@
 
         <!--Text view for showing the language name-->
         <TextView
-
             android:id="@+id/drink_name"
             android:layout_width="@dimen/textdrink_width"
             android:layout_height="wrap_content"
             android:layout_marginLeft="@dimen/textdrink_marginLeft"
             android:layout_marginTop="@dimen/textdrink_marginTop"
-
-            android:gravity="center_vertical|center_horizontal|top"
-            tools:text="text"
-            android:textSize="36sp"
+            android:textAppearance="@style/drink_card_beerName"
+            android:textAlignment="textStart"
+   
+            android:text="Titolo"
+    
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintLeft_toRightOf="@id/drink_image"
             app:layout_constraintTop_toTopOf="parent"
-            app:layout_constraintVertical_bias="0.048" />
+            app:layout_constraintVertical_bias="0.048"
+            />
 
         <!--Text View for showing the exp-->
+
         <Button
             android:id="@+id/button_go_to_detail"
-            android:layout_width="@dimen/gotodetail_width"
-            android:layout_height="@dimen/gototdetail_height"
-
-
-            android:layout_marginTop="12dp"
-            android:text="Vai al dettaglio"
+            android:layout_width="147dp"
```

Comment:
la dimensioni vanno specificate all'interno del file delle risorse e non utilizzate direttamente in modo hardcoded